src/main/webapp/user-page.html

```diff
@@ -88,6 +88,14 @@ <h2 style="inline-block">About me</h2>
     		<form id="message-form"action="/messages" method="POST" enctype="multipart/form-data" class="hidden">
       		<h3>Enter a new review:</h3>
     			<br/>
+          <select name="city">
+            <option value="">For which city?</option>
```

Comment:
Seems like this should not be a valid option. Instead, make it <option value="" disabled selected hidden> so that this is purely a placeholder. See https://stackoverflow.com/questions/5805059/how-do-i-make-a-placeholder-for-a-select-box

Reply:
Thanks! Just fixed it!